### PR TITLE
React 18 support

### PR DESCRIPTION
**Pull request by gunnartorfis**

### CODE REVIEW

**Comment by gunnartorfis** [comment]
Solves #365

File: packages/react-template-helper/react-template-helper.js [hunk 1]
```diff
@@ -5,7 +5,9 @@ checkNpmVersions({
 }, 'react-template-helper');
 
 const React = require('react');
+const { createRoot } = require('react-dom/client')
```
Comment (StorytellerCZ) [inline packages/react-template-helper/react-template-helper.js:8]: Won't this throw not found error on React < 18?

Comment (gunnartorfis) [reply packages/react-template-helper/react-template-helper.js:8]: Good spot, probably. 
Is there a good way to test the package locally with Meteor? Something similar to yarn link. 

Wouldn't feel comfortable merging anything without testing properly. I've only verified this is working on React 18.

Comment (StorytellerCZ) [reply packages/react-template-helper/react-template-helper.js:8]: I think it should be possible to move it bellow inside the function where you check for the React version.

Comment (gunnartorfis) [reply packages/react-template-helper/react-template-helper.js]: True! That works. I've pushed that fix.

**Review by StorytellerCZ — APPROVED** [review]

**Comment by donflopez** [comment]
can this be merged? I'm really looking forward to use React18 in my app.

**Comment by Grubba27** [comment]
Will publish a new version of react template helper

**Comment by Grubba27** [comment]
Screenshot 2022-11-21 at 10 09 38
published under the 0.2.15 code

**Pull request merged**